Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -848,35 +854,28 @@ defineSuite([
             expect(stats.numberOfPendingRequests).toEqual(1);
             scene.primitives.remove(tileset);
 
-            return root.contentReadyPromise.then(function(root) {
-                fail('should not resolve');
-            }).otherwise(function(error) {
-                // Expect the root to not have added any children from the external tileset.json
-                expect(root.children.length).toEqual(0);
-                expect(RequestScheduler.getNumberOfAvailableRequests()).toEqual(RequestScheduler.maximumRequests);
-            });
```

Comment:
The area I'm referring to is here : https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles-cache/Source/Scene/Batched3DModel3DTileContent.js#L142-L144. The previous check was waiting for the request to finish so that it could reach this area.